test/cppsuite/test_harness/timestamp_manager.h

```diff
@@ -50,11 +52,17 @@ class timestamp_manager : public component {
     load() override final
     {
         component::load();
-
-        _oldest_lag = _config->get_int(OLDEST_LAG);
+        int64_t oldest_lag = _config->get_int(OLDEST_LAG);
         testutil_assert(_oldest_lag >= 0);
```

Comment:
assert on oldest_lag instead ?

Reply:
We could directly do `_oldest_lag = _config->get_int(OLDEST_LAG);`

Reply:
Those asserts are definitely on the wrong thing, I wanted it to be explicit that get_int is returning an int64_t and our oldest and stable lag values should be uint64_ts.